web/src/app/users/UserEditDialog.tsx

```diff
@@ -171,30 +137,57 @@ function UserEditDialog(props: UserEditDialogProps): JSX.Element {
     errorList = [...errorList, ...handleValidation()]
 
     if (!errorList?.length && passwordChanged() && userHasBasicAuth) {
-      try {
-        await editBasicAuth()
-      } catch (err) {
-        console.error(err)
-        errorList = errorHandler(err, errorList)
-      }
+      await editBasicAuth(
+        {
+          input: {
+            userID: props.userID,
+            oldPassword: value.oldPassword || null,
+            password: value.password || null,
+          },
+        },
+        { additionalTypenames: ['UpdateBasicAuth'] },
+      ).then((result) => {
+        console.error(result.error)
```

Comment:
I think we can delete these console errors, when my password didn't match as a field error the console is printing undefined.

<img width="1034" alt="Screenshot 2024-01-08 at 10 59 23 AM" src="https://github.com/target/goalert/assets/11381794/049b3f5d-af51-4e3d-ac14-c27b41e3b9a8">